app/models/product.rb

```diff
@@ -15,6 +15,9 @@ class Product < ActiveRecord::Base
   has_many   :price_policies
   has_many   :training_requests, dependent: :destroy
 
+  cattr_accessor(:types) { ConfigurableArray.new([Instrument, Item, Service, Bundle]) }
```

Comment:
the idea of `ConfigurableArray` is to wrap up some somewhat obtuse insertion logic where you want to insert something before or after a certain element.

The ordering of Secure Rooms being before bundles is not critical, but I think it makes much more logical sense.